SU-HSDO/suhumsci
# SHS-5080: Paragraphs Browser hide filter

# READY FOR REVIEW

## Summary
Removes group filter and heading from paragraphs browser widget in Private and Flexible pages

## Steps to Test

1. Create a Flexible page
2. Click on `Add component`
3. Verify Filter label and field are not visible
4. Verify the "Other" category heading is not visible
5. Verify the search functionality displays the correct components
6. Repeat for Private page
7. Go to the flexible page `admin/structure/types/manage/hs_basic_page/form-display` and private page `/admin/structure/types/manage/hs_private_page/form-display` admin pages
8. Check that for flexible and private pages the fields containing the paragraph components have the `Show Group Label` and `Show Group Filter` check-boxes unchecked
## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
